packages/shared/logger.ts

```diff
@@ -15,3 +15,22 @@ const logger = winston.createLogger({
 });
 
 export default logger;
+
+export const authLogger = winston.createLogger({
```

Comment:
Given that we're writing to a file called `failedLogins`, I think this should indicate a failure instead of a generic `authFailureLogger`.
```suggestion
export const authLogger = winston.createLogger({
```